installing/installing_aws/ipi/installing-aws-multiarch-support.adoc

```diff
@@ -0,0 +1,79 @@
+:_mod-docs-content-type: ASSEMBLY
```

Comment:
This comment is for all three assemblies. I think you missed creating modules (?). From the `.Prerequisites` heading the content needs to be put in the individual procedure module.

Reply:
Updated as per the suggestion.